source/common/http/conn_manager_impl.cc

```diff
@@ -1272,6 +1278,11 @@ void ConnectionManagerImpl::ActiveStream::encodeData(ActiveStreamEncoderFilter*
 
   const bool trailers_exists_at_start = response_trailers_ != nullptr;
   for (; entry != encoder_filters_.end(); entry++) {
+    // If end_stream_ is marked for a filter, the data is not for this filter and filters after.
+    // Please refer to the `issue <https://github.com/envoyproxy/envoy/issues/5311>`_ for details.
```

Comment:
If you end up spelling out above, just refer the reader to the comment in the other function. You don't need to duplicate it.

Reply:
Done